src/test/java/org/opentripplanner/middleware/utils/ItineraryUtilsTest.java

```diff
@@ -0,0 +1,49 @@
+package org.opentripplanner.middleware.utils;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.opentripplanner.middleware.otp.response.Itinerary;
+import org.opentripplanner.middleware.otp.response.Leg;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+public class ItineraryUtilsTest {
+    @ParameterizedTest
+    @MethodSource("createItineraryHasTransitNoRentalsCases")
+    public void testItineraryHasTransitNoRentals(Itinerary itinerary, boolean expectedResult) {
+        Assertions.assertEquals(expectedResult, ItineraryUtils.itineraryHasTransitAndNoRentals(itinerary));
```

Comment:
It's much nicer to include a test case message so that it's easier to see in the test case output which test has failed if one does.

Reply:
Updated in https://github.com/ibi-group/otp-middleware/pull/91/commits/8cc7dd929604fe9ba216dcfe41d5ed59161da22c.